exercises/reforestation/canonical-data.json

```diff
@@ -0,0 +1,93 @@
+{
+    "exercise": "reforestation",
+    "version": "1.0.0",
+    "cases": [
+      {
+        "description": "Base case: empty tree",
+        "property": "treeTraversals",
+        "input": {
+          "preorder": [],
+          "inorder": []
+        },
+        "expected": {}
+      },
+      {
+        "description": "1 more step than base case",
+        "property": "treeTraversals",
+        "input": {
+            "preorder": ["a"],
+            "inorder": ["a"]
+        },
+        "expected": {"v":"a", "l": {}, "r": {}}
+      },
+      {
+        "description": "Core functionality with realistic size traversal",
+        "property": "treeTraversals",
+        "input": {
+            "preorder": ["a","i","x","f","r"],
+            "inorder": ["i","a","f","x","r"]
+        },
+        "expected": {"v": "a",
+            "l": {"v": "i", "l": {}, "r": {}},
+            "r": {"v": "x",
+                "l": {"v": "f", "l": {}, "r": {}},
+                "r": {"v": "r", "l": {}, "r": {}}
+            }
+        }
+      },
+      {
+        "description": "Traversals must be fully consumed. No extra values.",
+        "property": "treeTraversals",
+        "input": {
+          "preorder": ["a", "b"],
```

Comment:
There is a bit of inconsistency in the formatting of arrays. Here, `, ` is used to separate the elements of the array, whereas other test cases use `,`. I believe this version (that uses a space) is what most other exercises use, but I'm not completely certain.

Reply:
Ah. Linter didn't catch that, neither did I.